selene-lib/src/rules/limit_function_complexity.rs

```diff
@@ -0,0 +1,272 @@
+use super::*;
+use crate::ast_util::range;
+use std::convert::Infallible;
+
+
+use full_moon::{
+    ast::{self, Ast},
+    // node::Node,
+    visitors::Visitor,
+};
+
+use serde::Deserialize;
+
+#[derive(Clone, Copy, Deserialize)]
+pub struct LimitFunctionComplexityConfig {
+    maximum_complexity: u16,
+}
+
+impl Default for LimitFunctionComplexityConfig {
+    fn default() -> Self {
+        Self {
+            // eslint defaults to 20, but testing on OSS Lua shows that 20 is too aggressive
+            maximum_complexity: 40.to_owned(),

```

Comment:
the `to_owned()` here should not be necessary, 40 is Copy anyway. You probably saw this on a string, where `str.to_owned()` returns a String.